src/websockets/server.py

```diff
@@ -752,31 +794,33 @@ class Serve:
 
     def __init__(
```

Comment:
Suggest to use `WSSP = TypeVar('WSSP', bound=WebSocketServerProtocol)` here as well.

Reply:
See above — that didn't work well.